BurnOutSharp/PackerType/WinZipSFX.cs

```diff
@@ -60,8 +64,36 @@ private static string GetV3PlusVersion(byte[] fileContent, int position)
                "W.i.n.Z.i.p. .S.e.l.f.-.E.x.t.r.a.c.t.o.r." (Found in every 3+ version aside from 3.0.7158. This appears to be text for messages that WinZip SFX might display to a user, so much more text like it exists if an even more specfic check is needed for some reason.)
                Hex: 0x57, 0x00, 0x69, 0x00, 0x6E, 0x00, 0x5A, 0x00, 0x69, 0x00, 0x70, 0x00, 0x20, 0x00, 0x53, 0x00, 0x65, 0x00, 0x6C, 0x00, 0x66, 0x00, 0x2D, 0x00, 0x45, 0x00, 0x78, 0x00, 0x74, 0x00, 0x72, 0x00, 0x61, 0x00, 0x63, 0x00, 0x74, 0x00, 0x6F, 0x00, 0x72, 0x00 */
 
-            // "3.0.7158.0"
-            byte[] check = new byte[] { 0x33, 0x2E, 0x30, 0x2E, 0x37, 0x31, 0x35, 0x38, 0x2E, 0x30 };
+            // Find the end of the XML so it can be imported and parsed
+            // </assembly>
+            byte[] check = new byte[] { 0x3C, 0x2F, 0x61, 0x73, 0x73, 0x65, 0x6D, 0x62, 0x6C, 0x79, 0x3E };
+            if (fileContent.Contains(check, out int position3))
```

Comment:
Same here with naming. I only use generic names like `position` and `position2` in the normal checks because, well, they're generic. Within an extra method like this, feel free to name it something like `xmlEndPosition`.